apps/perms/api/user_permission/nodes.py

```diff
@@ -8,8 +8,7 @@
 from common.utils import get_logger, lazyproperty
 from perms import serializers
 from perms.utils import UserPermNodeUtil
-from .mixin import SelfOrPKUserMixin
-from .tree.mixin import RebuildTreeMixin
+from .mixin import SelfOrPKUserMixin, RebuildTreeMixin
 
 logger = get_logger(__name__)
 
```

Comment:
There are no significant changes from previous version to current as per given information. Therefore it is recommended to not use any modifications for this existing code.